src/main/java/com/kodilla/ecommercee/repository/GroupDao.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.Group;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+
+@Transactional
+@Repository
+public interface GroupDao extends CrudRepository<Group, Long> {
```

Comment:
Wszyscy uzywali klase z zakonczeniem Repository i proponuje zachowac tutaj konsekwencje. - GroupRepository

Reply:
Zgadzam się z Cezarym, nie jest to błąd, ale projekt będze czytelniejszy i bardziej estetyczny jesli zachowamy konsekwencje

Reply:
Nic dodać nic ująć.